mitodl/ocw-www
# give the search page its own header style

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
After merging https://github.com/mitodl/ocw-www/pull/67, the header was changed globally across the site.  This PR isolates search page styles to only apply to elements inside a div with the class `search-wrapper`.  The logo was changed back to the one with the transparent background.

#### How should this be manually tested?
 - Spin up the site and visit the home page.  You should see the header with the transparent background
 - Visit the search page and you should see the header with the black background

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/113049792-10dab500-9172-11eb-815c-b1313b020ec9.png)
![image](https://user-images.githubusercontent.com/12089658/113049802-13d5a580-9172-11eb-92da-06b5c34c031c.png)
![image](https://user-images.githubusercontent.com/12089658/113049820-17692c80-9172-11eb-99bd-93ae971f0631.png)
![image](https://user-images.githubusercontent.com/12089658/113049828-19cb8680-9172-11eb-9d02-3aa0b860f6bd.png)
